torchao/quantization/quant_primitives.py

```diff
@@ -95,9 +101,12 @@ def _get_and_check_qmin_qmax(dtype, quant_min, quant_max):
     verify that they are within the range of possible quant_min/quant_max
     for dtype
     """
-    if dtype not in _DTYPE_TO_QVALUE_BOUNDS:
+    if dtype in FP8_TYPES:
+        quant_min_lower_bound, quant_max_upper_bound = torch.finfo(dtype).min, torch.finfo(dtype).max
```

Comment:
we don't even need these if we don't want fp8 to have quant_min/quant_max I think

Reply:
We need it here
https://github.com/pytorch/ao/blob/efd480b43c14d0987e68834b501536d9f3330c68/torchao/quantization/quant_primitives.py#L270-L283

We don't want input for it. We can use default values.